quiz/quiz.sql

```diff
@@ -0,0 +1,6 @@
+create table quizo
+(
+    id serial primary key,
+    question text,
+    answer text
```

Comment:
Likewise, you might break this out into several tables:

- the `quiz` table could simply have a `title` (for a name like "Fall Midterm Exam")
- `questions` which contain the `text` and `answer` fields, as well as a `quiz_id` foreign key to the specific `quiz`
- `attempts` which contains a `question_id` foreign key to the `questions` table, an `answer` text field, and an `is_correct` boolean field.


The idea being that you could create a quiz, associate one or more questions with that quiz, and then a student supplies one attempt per question.

Continuing this idea, you could have a `students` table, and modify the `attempts` table so that it includes a `student_id` foreign key.